pages/_app.tsx

```diff
@@ -32,21 +32,21 @@ const WithStateContainerProviders = ({ children }: IProps) => (
         <EmpContract.Provider>
           <WethContract.Provider>
             <EmpState.Provider>
-              <EmpSponsors.Provider>
+              <Token.Provider>
                 <Collateral.Provider>
-                  <Token.Provider>
-                    <Totals.Provider>
-                      <PriceFeed.Provider>
+                  <PriceFeed.Provider>
```

Comment:
I had to reshuffle these around since `EMPSponsors` now depends upon `Token` and `PriceFeed`